North-Seattle-College/ad440-winter2022-thursday-repo
# add frontend folder and initial react files

Closes #11 - Create a folder for the frontend and checkin the initial React application files

## Description of Change

Creates a "frontend" folder and the initial react files within the folder. 

## Prequisites

- Node version of at least 10
- Npm version of at least 5.2

## How to test
- Clone the repo https://github.com/North-Seattle-College/ad440-winter2022-thursday-repo
- Cd into ad440-winter2022-thursday-repo folder
- Checkout frontend-folder-initial-react-files-issue11 branch
- Cd into frontend folder 
- Run npm install
- Run npm start
- The app should automatically start in your web browser with URL http://localhost:3000/ and "Floop App" on the window.

<img width="567" alt="Screen Shot 2022-01-20 at 4 06 17 PM" src="https://user-images.githubusercontent.com/47171828/150442051-5489f119-8c8c-4f43-89ed-055f7bfb4826.png">
